src/mam/mod_mam_utils.erl

```diff
@@ -427,10 +427,10 @@ retracted_element(#{retract_on := stanza_id,
                        MaybeOriginId
                       ]}.
 
-maybe_append_origin_id(#{origin_id := <<>>}) ->
```

Comment:
you can add a typespec for this function

Reply:
Done